mirrord/config/src/fs/advanced.rs

```diff
@@ -14,29 +14,36 @@ use crate::{
 ///
 /// Allows the user to specify:
 ///
-/// - `MIRRORD_FILE_OPS` and `MIRRORD_FILE_RO_OPS`;
-/// - `MIRRORD_FILE_FILTER_INCLUDE` and `MIRRORD_FILE_FILTER_EXCLUDE`;
-///
+/// What is the default behavior for file operations:
+/// 1. read - Read from the remote file system (default)
+/// 2. write - Read/Write from the remote file system.
+/// 3. local - Read from the local file system.
+/// 4. disabled - Disable file operations.
+/// 
+/// Besides the default behavior, user can specify behavior for specific regex patterns. Case insensitive.
```

Comment:
Why be case insensitive?

Reply:
If we're letting the user pick their own regexes, then I think the least surprising thing would be for us to do exactly what they said (if they want case insensitive then they specify as `/etc/[lL]ala`).

I say this because I can imagine this being annoying if i wanted to differentiate between both, and saw that the tool doesn't.